internal/config/config.go

```diff
@@ -18,15 +18,24 @@ type Config struct {
 	KubeconfigPath string        `mapstructure:"kubeconfigPath"`
 	NodeIP         string        `mapstructure:"nodeIP"`
 	NodeName       string        `mapstructure:"nodeName"`
-	HTTPServerPort string        `mapstructure:"httpServerPort"`
 	Interval       time.Duration `mapstructure:"interval"`
-	Timeout        time.Duration `mapstructure:"timeout"`
+	Timeout        time.Duration `mapstructure:"timeout"` // TODO: Unimplemented/unused (issue #322)
+
+	Sink struct {
```

Comment:
I would avoid inline struct (merely because the file is not using them)

Reply:
mmm but it is true that it has only an attribute... I am not sure, I still prefer them not inlined, but I understand why you place it there, up to you if you want to change it :)

Reply:
I just copy and pasted what @roobre had before me for the KSM as it is a single attribute. KSM has 2:
```go
	Discovery   struct {
		// Wait BackoffDelay between discovery attempts.
		BackoffDelay time.Duration `mapstructure:"backoffDelay"`
		// Give up discovery and fail if Timeout has passed since first attempt.
		Timeout time.Duration `mapstructure:"timeout"`
	} `mapstructure:"discovery"`
```

I prefer to not complicate this more as there is only one sink (yet and hoping to not have more than one)